### PR TITLE
Fix Textarea Height in Comments

### PR DESCRIPTION
**Pull request by harpreet9629**
I got this issue in Firefox Browser

Before
Screenshot_2

After
Screenshot_3

### CODE REVIEW

File: src/styles.scss [hunk 1]
```diff
@@ -1020,11 +1020,16 @@ $green-font-color: #19B028;
...
+.feed-create-comment__textarea{
+  min-height: 55px;
+}
+
 .feed-create-post__textarea:focus {
   border: none;
   box-shadow: none;
 }
 
+
```
Comment (lazynina) [inline src/styles.scss:1032]:
```suggestion
```

File: src/styles.scss [hunk 2]
```diff
@@ -1020,11 +1020,16 @@ $green-font-color: #19B028;
   background-image: url('/assets/img/default_profile_pic.png');
 }
 
+.feed-create-comment__textarea{
```
Comment (lazynina) [inline src/styles.scss:1023]:
```suggestion
.feed-create-comment__textarea {
```

**Review by lazynina — CHANGES_REQUESTED** [review]
How can I reproduce the small textarea seen in the screenshot above? This change seems fine, but I'm not able to reproduce the issue.

**Comment by harpreet9629** [comment]
I got this issue in Firefox, Mostly i use Firebox. Its fine in Chrome browser, this issue happen with autoresize.

**Review by lazynina — APPROVED** [review]

**Comment by lazynina** [comment]
> I got this issue in Firefox, Mostly i use Firebox. Its fine in Chrome browser, this issue happen with autoresize.

yep was able to reproduce after my initial review -- good find!

**Pull request merged**